scripts/quests/bastok/Return_to_the_Depths.lua

```diff
@@ -0,0 +1,256 @@
+-----------------------------------
+-- Return to the Depths
+-- Ayame
+-- Log ID = 1
+-- Quest ID = 78
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require("scripts/globals/status")
+require('scripts/globals/titles')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.BASTOK, xi.quest.id.bastok.RETURN_TO_THE_DEPTHS)
+
+quest.reward =
+{
+    fame     = 60,
+    fameArea = xi.quest.fame_area.BASTOK,
+    item     = xi.items.BOWYER_RING,
+    title    = xi.title.GOBLIN_IN_DISGUISE,
+}
+
+quest.sections =
+{
+    -- Available
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:getFameLevel(xi.quest.fame_area.BASTOK) >= 5 and
+            player:hasCompletedQuest(xi.quest.log_id.BASTOK, xi.quest.id.bastok.A_QUESTION_OF_FAITH)
+        end,
+
+        [xi.zone.METALWORKS] =
+        {
+            ['Ayame'] = quest:progressEvent(879),
+
+            onEventFinish =
+            {
+                [879] = function(player, csid, option, npc)
+                        quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    -- Accepted
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.METALWORKS] =
+        {
+            ['Ayame'] =
+            {
+                onTrigger = function(player, csid, option, npc)
+                    if quest:getVar(player, 'prog') == 10 then
+                        return quest:progressEvent(881)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [881] = function(player, csid, option, npc)
+                    quest:complete(player)
+                end,
+            },
+        },
+
+        [xi.zone.OLDTON_MOVALPOLOS] =
+        {
+            ['Tarnotik'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'prog') == 6 then
+                        return quest:progressEvent(42, 11, 23, 2963, -70500718, xi.items.BULB_OF_MISAREAUX_GARLIC, 0, 2303)
```

Comment:
This should also work. Might need to test it, but it should be fine!
```suggestion
                        return quest:progressEvent(42, 0, 0, 0, 0, xi.items.BULB_OF_MISAREAUX_GARLIC)
```

Reply:
Yeah, I wasn't sure how the extra params played out, so I just left the captured ones in there.  Let me see what they'll do.

Reply:
Yep, seems to work!  Fixed